servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -148,6 +148,7 @@ quarkus.index-dependency.protobuf.group-id=com.google.protobuf
 quarkus.index-dependency.protobuf.artifact-id=protobuf-java
 quarkus.native.additional-build-args=\
   --rerun-class-initialization-at-runtime=org.rocksdb.RocksDB,\
+  --rerun-class-initialization-at-runtime=org.apache.http.impl.auth.NTLMEngineImpl,\
```

Comment:
Why do we need to add NTLM?

Reply:
I tried to investigate why, but couldn't figure out the actual cause.
But that class initialization triggers some `java.util.Random` initialized in a static final field.